app/models/votable_field.rb

```diff
@@ -0,0 +1,18 @@
+class VotableField < ActiveRecord::Base
+  belongs_to :conference
+  validates :title, :votable_type, presence: true
+
+  # ratyrate does not allow criterias to have spaces in them
+  validate :no_spaces_in_title
```

Comment:
By proper names you mean names without spaces, the way it is currently done, right? :slightly_smiling_face: